pkg/tailfile/fuzz_test.go

```diff
@@ -0,0 +1,38 @@
+package tailfile
+
+import (
+	"os"
+	"testing"
+
+	fuzz "github.com/AdaLogics/go-fuzz-headers"
+)
+
+func FuzzTailfile(f *testing.F) {
+	f.Fuzz(func(t *testing.T, data []byte) {
+		if len(data) < 5 {
+			return
+		}
+		ff := fuzz.NewConsumer(data)
+		n, err := ff.GetUint64()
+		if err != nil {
+			return
+		}
+		fileBytes, err := ff.GetBytes()
+		if err != nil {
+			return
+		}
+		defer os.Remove("tailfile")
+		fil, err := os.Create("tailfile")
```

Comment:
Oh, this one probably as well (cleaning up the actual file will be taken care of by go cleaning up the testDir it's in);

```suggestion
		tempDir := t.TempDir()
		fil, err := os.Create(filepath.Join(tempDir, "tailFile"))
```